doc/api/events.md

```diff
@@ -223,6 +279,16 @@ changes:
 
 The `'removeListener'` event is emitted *after* the `listener` is removed.
 
+### Event: 'rejection'
+<!-- YAML
+added: REPLACEME
+-->
+
+* `err` The error
+
+The `'rejection'` event is emitted if a promise rejection happens and
```

Comment:
It may be helpful for the rejection event to include the name of the event that triggered it.

```js
ee.on('foo', async () => {})
ee.on('bar', async () => {})
ee.on('rejection', async () => {
  // Which one rejected?
})
```

Also, what happens if I provide an async handler to `'error'` or `'rejection'` and those reject? Will I end up with a loop?

A rejection in the rejection handler should always emit an error.

A rejection in the error handler should always cause an unhandledRejection or unhandledException

Reply:
This is fixed, please check!

Reply:
How about the rejection call including the event that failed?

Reply:
> How about the rejection call including the event that failed?

This follows the same convention of `'error'`. I think we should keep consistency.